File: noi.openjudge.cn/ch0104/16.cpp
// http://noi.openjudge.cn/ch0104/solution/39178647/ 

#include<iostream>
using namespace std;
int main() {
    int a,b,c;
    cin >> a >> b >> c;
    if (a + b > c) {
        if (a + c > b) {
            if (b + c > a) {
                cout << "yes" << endl;
            } else {
                cout << "no" << endl;
            }
        } else {
            cout << "no" << endl;
        }
    } else {
        cout << "no" << endl;
    }
}

File: noi.openjudge.cn/ch0105/16.py
// http://noi.openjudge.cn/ch0105/solution/39419626/

#include<iostream>
using namespace std;
int main() {
    int n,a,m = 0;
    double x = 50,y = 200;
    cin >> n >> a;
    bool flag = false;
    for (int i=1;i<=20;i++) {
        x = i * n;
        if (x + 1e-8 > y) {
            cout << i << endl;
            flag = true;
            break;
        }
        y *= 1 + a / 100.0;
    }
    if (flag == false) {
        cout << "Impossible" << endl;
    }
}
